.github/workflows/cicd.yml

```diff
@@ -0,0 +1,63 @@
+name: DynoDM CI/CD
+on: push
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: build node modules
+        run: yarn
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ github.sha }}
```

Comment:
Maybe this could be based on the package.json sha instead? 🙂 Also don't we need to get the cache before running the `yarn` command? 🙂

Reply:
removed caching :)